rules-threat-hunting/windows/process_creation/proc_creation_win_iexpress_execution.yml

```diff
@@ -0,0 +1,36 @@
+title: New Self Extracting Package Created Via IExpress.EXE
+id: c2b478fc-09bf-40b2-8768-ab3ec8d61c9a
+status: experimental
+description: |
+    Detects the "iexpress.exe" utility in order to create self-extracting packages.
+    Attackers where seen leveraging "iexpress" in to compile packages on the fly via ".sed" files.
+    Investigate the command line options provided to "iexpress" and in case of a ".sed" file, check the contents and legitimacy of it.
```

Comment:
```suggestion
    Detects the "iexpress.exe" utility creating self-extracting packages.
    Attackers where seen leveraging "iexpress" to compile packages on the fly via ".sed" files.
    Investigate the command line options provided to "iexpress" and in case of a ".sed" file, check the contents and legitimacy of it.
```